frontend/src/concepts/roleBinding/RoleBindingPermissionsNameInput.tsx

```diff
@@ -52,10 +52,6 @@ const RoleBindingPermissionsNameInput: React.FC<RoleBindingPermissionsNameInputP
     return { value: displayName, content: displayName };
   });
 
-  if (value && !typeAhead.includes(value)) {
-    selectOptions.push({ value, content: value });
-  }
-
```

Comment:
Removed the code that was adding the current value to `selectOptions` if it wasn't in `typeAhead`. This ensures that custom items don't persist in the dropdown.

Reply:
Ah fair point, I guess removing this entirely solves that as well